HIP-Basic/vulkan_interop/README.md

```diff
@@ -54,10 +54,60 @@ To signal a shared semaphore in HIP, the `hipSignalExternalSemaphoresAsync` func
+          </ClCompile>
+          <Link>
+            ...
+            <AdditionalDependencies>glfw3dll.lib;<other_dependencies>;%(AdditionalDependencies)</AdditionalDependencies>
+            <AdditionalLibraryDirectories>$(GLFW_DIR)\lib;<other_library_directories><AdditionalLibraryDirectories>
+            ...
+          </Link>
+        </ItemDefinitionGroup>
+        ```
+- Vulkan headers. On Linux, the vulkan headers can be directly obtained from some package managers:
+    - Linux
+        - APT: `apt-get install -y libvulkan-dev`
+        - Pacman: `pacman -S vulkan-headers vulkan-icd-loader`
+        - DNF: `dnf install vulkan-headers vulkan-icd-loader`
+
+    But they may be as well obtained by installing the [LunarG Vulkan SDK](https://vulkan.lunarg.com/). CMake will be able to find the SDK using the `VULKAN_SDK` environment variable, which is set by default using the SDK activation script.
+
+    On Windows, on the other hand, the headers can only be obtained from the [LunarG Vulkan SDK](https://vulkan.lunarg.com/). Contrary to Unix-based OSs, the `VULKAN_SDK` environment variable is not automatically provided on Windows, and so it should be set to the appropriate path before invoking CMake.
+
+    Note that `libvulkan` is _not_ required, as the example loads function pointers dynamically.
+
+- Validation layers. The `VK_LAYER_KHRONOS_validation` layer is active by default to perform general checks on Vulkan, thus the [Khronos' Vulkan Validation Layers (VVL)](https://github.com/KhronosGroup/Vulkan-ValidationLayers/tree/main#vulkan-validation-layers-vvl) will need to be installed on the system if such checks are desirable. It can be either installed from a package manager (on Linux), built and configured from source or installed as part of the [LunarG Vulkan SDK](https://vulkan.lunarg.com/).
+Package managers offering the validation layers package include:
+    - APT: `apt install vulkan-validationlayers-dev`
+    - Pacman: `pacman -S vulkan-validation-layers`. Note that with pacman both the validation layers and headers (among others) can be also installed with `pacman -S vulkan-devel`.
+    - DNF: `dnf install vulkan-validation-layers`
+
+    For the second approach, build instructions are provided on [Khronos Vulkan-ValidationLayers repository](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/main/BUILD.md) and Vulkan's [Layers Overwiew and Configuration](https://vulkan.lunarg.com/doc/view/latest/windows/layer_configuration.html) document offers several approaches for its configuration.
+
+- `glslangValidator`. It is used in the example as a shader validation tool. It may be installed by [building manually from source](https://github.com/KhronosGroup/glslang#building-cmake), by downloading the binaries for the corresponding platform directly from the [main-tot](https://github.com/KhronosGroup/glslang/releases/tag/main-tot) release on GitHub or installed as part of the [LunarG Vulkan SDK](https://vulkan.lunarg.com/).
```

Comment:
Great change to the documentation. Thank you for calling out the dependencies. 
re: glslangvalidator, you can install it via package manager too:  `sudo apt install glslang-tools`

Reply:
Oh, missed that one, nice catch :). I added the change to the general fixes PR (https://github.com/ROCm/rocm-examples/pull/98) and rebased this one.